src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,91 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa_add_support_to_verify_authentication_indicators(client: Client, ipa: IPA):
+    """
+
+    Kerberos client may have different means to prove possession of a client principal
+    credentials to a KDC. There are several so-called ‘pre-authentication mechanisms’ that are
+    used for this purpose. FreeIPA KDC is able to record which pre-authentication method was used
+    when issuing the ticket granting ticket. The specific label is called an authentication indicator.
+    https://freeipa.readthedocs.io/en/latest/workshop/11-kerberos-ticket-policy.html
+
+    :title: Add support to verify authentication indicators in pam_sss_gss
```

Comment:
So this is not a general test, this only tests against pam_sss_gss, it should be renamed to include that in the test name, that it isn't a test indicator's test, should we do one without gss and see if the there is any log messages?